votes/0547-andrewdi-as-tidb-reviewer.md

```diff
@@ -0,0 +1,27 @@
+# A Vote for AndrewDi as TiDB committer
+
+## Proposal
+
+[@AndrewDi](https://github.com/AndrewDi) has been working on `pingcap/tidb` for several years. He has contributed a lot of bugfixes/enhancements for TiDB, among which the most noticable one is that he is the author of 'View' in TiDB. Also, AndrewDi deeply participated in the design of 'Sequence' in TiDB.
+
+Here lists some details of his contributions:
+
+* [Authored pull requests](https://github.com/pingcap/tidb/pulls?q=is%3Apr+author%3Aandrewdi+is%3Aclosed).
+* [Reviewed pull requests](https://github.com/pingcap/tidb/pulls?q=is%3Apr+reviewed-by%3AAndrewDi)
+
+I (@bb7133) hereby nominate @AndrewDi as TiDB committer and call for a vote for the following reasons:
+
+* His contributions covered a varity of modules of TiDB-Server(expression, executor, planner, DDL...)
+* He is the one who introduce the 'View' support to TiDB, which is valuable for TiDB as a database.
+
+## Deadline
+
+The vote will be open for at least 2 days unless there is an objection or not enough votes.
```

Comment:
```suggestion
The vote will be open for at least 6 days unless there is an objection or not enough votes.
```

according to https://github.com/pingcap/community/blob/master/teams/README.md#actions

Reply:
Fixed, sorry for the mistake!